SPECS/python-apache-libcloud/python-apache-libcloud.spec

```diff
@@ -0,0 +1,1064 @@
+#
+# spec file for package python-apache-libcloud
+#
+# Copyright (c) 2022 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+%{?!python_module:%define python_module() python-%{**} python3-%{**}}
+Name:           python-apache-libcloud
+Version:        3.5.1
+Release:        2
+Summary:        Abstraction over multiple cloud provider APIs
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages/Python
+URL:            https://libcloud.apache.org
+Source0:        http://archive.apache.org/dist/libcloud/apache-libcloud-%{version}.tar.bz2#/apache-libcloud-%{version}.tar.gz
+Patch1:         gce_image_projects.patch
+Patch2:         ec2_create_node.patch
+Patch3:         skip-some-tests-for-older-paramiko-versions.patch
+# PATCH-FIX-UPSTREAM https://github.com/Kami/libcloud/commit/e62bb28cdbd685203d44a9a4028f311ea155476c Use unittest.mock library from stdlib instead of using 3rd party mock dependency.
+Patch4:         mock.patch
+BuildRequires:  libvirt-python3
+BuildRequires:  python3-lockfile
+BuildRequires:  python3-lxml
+BuildRequires:  python3-paramiko
+BuildRequires:  python3-pyOpenSSL
+BuildRequires:  python3-pytest
+BuildRequires:  python3-requests-mock
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-typing
+BuildRequires:  python3-xml
+BuildRequires:  fdupes
+BuildRequires:  pyproject-rpm-macros
+Requires:       python3-lxml
+Requires:       python3-requests
+Requires:       python3-typing
+Suggests:       python-libvirt-python
+Suggests:       python-lockfile
+Suggests:       python-paramiko
+Suggests:       python-pysphere
```

Comment:
We no longer ship RPMs that don't have a python major version number as part of their name

Reply:
Removed.